packages/datatrak-web-server/src/utils/processSurveyResponse.ts

```diff
@@ -105,6 +105,17 @@ export const processSurveyResponse = async (
         surveyResponse.entity_id = entityId;
         break;
       }
+      case QuestionType.File: {
+        const { name, value } = answer as FileUploadAnswerT;
+        answersToSubmit.push({
+          ...answerObject,
+          body: {
+            data: value,
+            uniqueFileName: getUniqueSurveyQuestionFileName(name),
```

Comment:
Hey @rohan-bes I decided to do this in the orchestration server, as opposed to the front-end app because we need the FE answers to be a particular shape